scripts/perf/vm_scripts/centaur/centaur.conf

```diff
@@ -1,3 +1,5 @@
+akka.http.client.parsing.max-content-length = 524288000
```

Comment:
Hmm Centaur by default pulls the entire metadata from Cromwell. I added a configuration option in Centaur to use the include field and reduce the amount but it's empty by default and was to avoid cases where Cromwell is not even able to generate the metadata. We could use it here but we need to determine what we want to pull out from metadata in advance, which depends on what assertion we want to make, which I'm unsure about 😄